ci/tests/plugin-compiler/test.sh

```diff
@@ -29,7 +29,7 @@ docker run --rm -v `pwd`/testplugin:/plugin-source tykio/tyk-plugin-compiler:${t
 # This ensures correct paths when running by hand
 TYK_GW_PATH=$(readlink -f $(dirname $(readlink -f $0))/../../..)
 # Get version from source code (will not include rc tags - same as ci/images/plugin-compiler build.sh)
-TYK_GW_VERSION=$(perl -n -e'/v(\d+).(\d+).(\d+)/'' && print "v$1\.$2\.$3"' $TYK_GW_PATH/gateway/version.go)
+TYK_GW_VERSION=$(perl -n -e'/v(\d+).(\d+).(\d+)/'' && print "v$1\.$2\.$3"' $TYK_GW_PATH/internal/build/version.go)
```

Comment:
regarding this way of getting `TYK_GW_VERSION` - would the version variable in version.go be updated in source code?

Reply:
No, it is injected and "tyk version" should be used when the binary is available. We can add a -clean flag to provide a sanitized form for CI job uses.